kymatio/keras.py

```diff
@@ -0,0 +1,26 @@
+from tensorflow.keras.layers import Layer
+from tensorflow.python.framework import tensor_shape
+from .tensorflow import Scattering2D as scattering2D
+
+class Scattering2D(Layer):
+    def __init__(self,  J, shape, L=8, max_order=2, pre_pad=False):
+        self.J = J
+        self.shape = shape
+        self.L = L
+        self.max_order = max_order
+        self.pre_pad = pre_pad
+        super(Scattering2D, self).__init__()
+
+    def call(self, x):
+        S = scattering2D(J=self.J, shape=self.shape, L=self.L,
```

Comment:
if I read this correctly, it will create the filters at every call. maybe put that into a build function?

Reply:
Good point - will test to see if it makes a difference